packages/common/src/types/auth.ts

```diff
@@ -80,8 +80,12 @@ export type UserAccessControls = {
 };
 
 export type DashboardAccess = {
-    /** The dashboard ID the account has access to */
-    dashboardId: string;
+    /** The content ID the account has access to (dashboard or chart) */
+    contentId?: string;
+    /** The type of content (dashboard or chart) */
+    contentType?: 'dashboard' | 'chart';
+    /** @deprecated Use contentId instead. The dashboard ID the account has access to */
+    dashboardId?: string;
```

Comment:
Do we need to deprecate since this object is only used internally? I wonder if we could create a new `content` object with `content: { type, id|idOrSlug }`?